apps/web/components/booking/AvailableTimes.tsx

```diff
@@ -83,7 +83,7 @@ const AvailableTimes: FC<AvailableTimesProps> = ({
               pathname: router.pathname.endsWith("/embed") ? "../book" : "book",
               query: {
                 ...router.query,
-                date: dayjs(slot.time).format(),
+                date: dayjs.utc(slot.time).tz(timeZone()).format(),
```

Comment:
slot.time comes in as UTC, and dayjs() automatically converted this to browser time. We don't want that, we want to submit to user preferences.